adapters/repos/db/docid/scan.go

```diff
@@ -86,14 +86,12 @@ func (os *objectScannerLSM) scan() error {
 		propStrings[i] = []string{os.properties[i]}
 	}
 
-	// The typed properties are needed for extraction from json
-	var properties models.PropertySchema
-	propertiesTyped := map[string]interface{}{}
-
-	for _, prop := range os.properties {
-		propertiesTyped[prop] = nil
-	}
```

Comment:
I did not understand the usage of this thing as `UnmarshalPropertiesFromObject` immediately clears the map again

Reply:
maybe it was there to ensure that we have all properties in the map, including those empty ones?

Reply:
But we clear the map immediately anyway, this is probably a leftover from some refactoring

![image](https://github.com/user-attachments/assets/ae4aaa9e-8864-41f8-93c8-e1b90c64579e)
![image](https://github.com/user-attachments/assets/3e116be0-2194-47de-8d96-18552d48dcd5)

Reply:
wow, didn't know that, nice catch